src/AdminPanel/modules/Login.tsx

```diff
@@ -0,0 +1,3 @@
+const Login = () => <div>Login page</div>;
+
+export default Login;
```

Comment:
also please nest this file a bit more, create LoginPage directory and put index.ts for exports in it, just like we spoke